docs/Administrator Guide/Accessing Gluster from Windows.md

```diff
@@ -58,7 +58,8 @@ The servers may/may not be part of the trusted storage pool. Preferable number o
    192.168.1.20/24 eth0
    192.168.1.21/24 eth0
    ```
- 7. If SELinux is enabled and enforcing, try the following command if ctdb fails.  
+ 7. Uncomment	`CTDB_SAMBA_SKIP_SHARE_CHECK=yes` in /etc/ctdb/script.options to disable checking of the shares by ctdb
```

Comment:
Seems to be on the right lines. What about the following variant?
> 7. Either uncomment `CTDB_SAMBA_SKIP_SHARE_CHECK=yes` or add `CTDB_SAMBA_SKIP_SHARE_CHECK=yes` in its absence inside /etc/ctdb/script.options to disable checking of the shares by ctdb.

Reply:
done